src/pipx/constants.py

```diff
@@ -4,20 +4,31 @@
 from pathlib import Path
 from textwrap import dedent
 from typing import NewType, Optional
+from platformdirs import user_cache_path, user_data_path, user_log_path
 
 DEFAULT_PIPX_HOME = Path.home() / ".local/pipx"
 DEFAULT_PIPX_BIN_DIR = Path.home() / ".local/bin"
```

Comment:
This isn't the default any more, is it? Can you change the fallback mechanism?

Reply:
What do you suggest? Change it to `user_data_path("pipx")` or remove this constant?

Reply:
> Change it to `user_data_path("pipx")`

That would be my suggestion.